src/data/code-samples/en/input-errors/swift-ui.md

```diff
@@ -0,0 +1,34 @@
+# Input errors - SwiftUI
+
+In SwiftUI, we recommend using a [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to indicate an error. The error message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
```

Comment:
remove "a" after "using" because views is plural.
or make views singular (view)